--- origsrc/clamtk-3.11/clamtk	2008-08-28 01:07:38.526944600 -0500
+++ src/clamtk-3.11/clamtk	2008-08-28 01:07:12.167569600 -0500
@@ -60,7 +60,7 @@
 my @h_files = ();
 my $h_label;
 
-if ( $> == 0 ) {
+if ( $> == 0 || $^O == 'cygwin' ) {
     $authenticate = 'gtk-yes';
     $a_tooltip    = gettext("Check for signature updates");
 }
@@ -1112,7 +1112,7 @@
 }
 
 sub update {
-    if ( $> != 0 ) {
+    if ( $> != 0 && $^O != 'cygwin' ) {
         show_message_dialog( $window, 'info', 'close',
             gettext("You must be root to install updates.") );
         return;
